.github/workflows/destroy.yml

```diff
@@ -0,0 +1,62 @@
+name: Destroy
+
+on:
+  pull_request:
+    branches: [ master ]
+    types: [closed]
+
+env:
+  CF_PROVIDER_VERSION: 0.12.3
+  DOCKERHUB_REPOSITORY: dfedigital/teaching-vacancies
+  PR_NAME: ${{ format('pr-{0}', github.event.number) }}  
+
+jobs:
+  deploy:
+    name: Destroy review app
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      name: Checkout Code
+
+    - name: Terraform pin version
+      uses: hashicorp/setup-terraform@v1
+      with:
+        terraform_version: 0.13.1
+
+    - name: Set environment variables
+      id: set_env_vars
+      run: |
+        TAG=review-${PR_NAME}-${GITHUB_SHA}-$(date '+%Y%m%d%H%M%S')
+        echo ::set-env name=TAG::${TAG}
+        echo ::set-env name=CF_PROVIDER_BINARY::terraform-provider-cloudfoundry_v${CF_PROVIDER_VERSION}_linux_amd64
+        echo ::set-env name=CF_PROVIDER_URL::https://github.com/cloudfoundry-community/terraform-provider-cloudfoundry/releases/download/v${CF_PROVIDER_VERSION}
+        echo ::set-env name=DOCKER_BUILDKIT::1
+
+    - name: Terraform install CloudFoundry provider
+      run: |
+        CF_PROVIDER_DIR=$HOME/.local/share/terraform/plugins/terraform.implied.local.mirror/cloudfoundrylocal/cloudfoundry/${CF_PROVIDER_VERSION}/linux_amd64
+        mkdir -p ${CF_PROVIDER_DIR}
+        wget -O ${CF_PROVIDER_DIR}/${CF_PROVIDER_BINARY} ${CF_PROVIDER_URL}/${CF_PROVIDER_BINARY}
+        chmod +x ${CF_PROVIDER_DIR}/${CF_PROVIDER_BINARY}
+        echo "Content of directory ${CF_PROVIDER_DIR}:"
+        cd ${CF_PROVIDER_DIR} && pwd
+        ls -l
+
+    - name: Terraform fmt check
```

Comment:
This is not required for destroy

Reply:
Moved `terraform fmt` to the Lint workflow